docker-app/qfieldcloud/core/models.py

```diff
@@ -1546,6 +1546,36 @@ def fallback_output(self) -> str:
                 "The job ended in unknown state. Please verify the project is configured properly, try again and contact QFieldCloud support for more information."
             )
 
+    def clean(self) -> None:
+        """
+        Prevent creating new jobs if the user is inactive, over quota
+        or the project has online vector layers (postgis) and his account does not support it
+        """
+        useraccount = self.created_by.useraccount
+        current_subscription = useraccount.active_subscription
+
+        if not current_subscription.is_active:
+            raise ValidationError(_("Cannot create job for inactive user."))
```

Comment:
```suggestion
        if not current_subscription.is_active:
            raise ValidationError(_("Cannot create job for user with inactive subscription."))
```

Just a slight wording improvement, because now it reads like `user.is_active` has been checked.